core/blockchain.go

```diff
@@ -2051,13 +2054,21 @@ func (bc *BlockChain) insertChain(chain types.Blocks, verifySeals bool) (int, er
 
 		// Enable prefetching to pull in trie node paths while processing transactions
 		statedb.StartPrefetcher("chain")
-
+		var followupInterrupt uint32
```

Comment:
Should we set `followupInterrupt` to 1 before the next loop starts? To avoid doing anything tricky to cause prefetching to be slower than block execution.

Reply:
good idea.

Reply:
please check the latest commit